python/tvm/autotvm/tophub.py

```diff
@@ -36,13 +36,13 @@
 
 # the version of each package
 PACKAGE_VERSION = {
-    'arm_cpu': "v0.04",
+    'arm_cpu': "v0.05",
     'llvm':    "v0.03",
 
-    'cuda':    "v0.04",
+    'cuda':    "v0.05",
```

Comment:
CI test with "fail to download tophub package". Have the TOPHUB schedules for cuda been updated?

Reply:
see http://ci.tvm.ai:8080/blue/organizations/jenkins/tvm/detail/PR-3642/20/pipeline/240